lib/net/packets.js

```diff
@@ -957,6 +970,663 @@ class NotFoundPacket extends InvPacket {
   }
 }
 
+/**
+ * GetCFilters Packet
+ * @extends Packet
+ * @property {Number} startHeight
+ * @property {Hash} stopHeight
+ * @property {Number} filterType
+ */
+
+class GetCFiltersPacket extends Packet {
+  /**
+   * Create a `getcfilters` packet.
+   * @param {Number} filterType - Filter type.
+   * @param {Number} startHeight - Start block height.
+   * @param {Hash} stopHash - Stop block hash.
+   */
+
+  constructor(filterType, startHeight, stopHash) {
+    super();
+
```

Comment:
let's just add this here, and for all the filterType's, since there is no default value

```js
    assert(typeof filterType === 'number');
```

Reply:
adding this will create a problem while capturing the packets from the peers as we do not pass any parameters to the static from raw function and thus the filterType is undefined.